style.css

```diff
@@ -0,0 +1,653 @@
+* {
+  padding: 0;
+  margin: 0;
+  font-family: 'lato', sans-serif;
+  box-sizing: border-box;
+}
+
+body {
+  background-color: #e5e5e5;
```

Comment:
You're using some nice colors but make sure to also use the 🎨 colors mentioned in the colors swatch, take a look below 👇🏽

![Screenshot 2023-04-18 at 22 31 14](https://user-images.githubusercontent.com/45739963/232951351-82a75f23-9f8e-419b-8d2b-b60795cd1cd3.png)